client/components/PubCollectionsListing/PubCollectionsListing.tsx

```diff
@@ -29,6 +28,9 @@ type Props = {
 	renderTriggerButtonForQueryList?: () => React.ReactNode;
 	renderDragElementInPortal?: boolean;
 };
+type TagToCreate = {
+	tagNameToCreate: string;
```

Comment:
This should actually be `tagTitleToCreate` since Collections have "titles" not "names". Sorry for the misdirection!